config/config.go

```diff
@@ -307,6 +333,20 @@ func NewAgentConfig(agentIni *File, agentYaml *YamlAgentConfig) (*AgentConfig, e
 	return cfg, nil
 }
 
+// Compile each word in the list into a regex pattern to match against the cmdline arguments
+func CompileStringsToRegex(words []string) []*regexp.Regexp {
+	compiledRegexps := make([]*regexp.Regexp, 0, len(words))
+	for _, word := range words {
+		pattern := `(-{1,2}` + word + `[^= ]*[ =])([^ \n]*)`
+		r, err := regexp.Compile(pattern)
```

Comment:
You should be able to use: `regexp.MustCompile`. I think it's better to fail fast on invalid regexps than silently ignoring them (especially if a user thinks their sensitive data is actually being blacklisted)